haiku-apps/bescreencapture/bescreencapture-2.3.2.recipe

```diff
@@ -11,13 +11,12 @@ user to launch BeScreenCapture and start/stop recording using a keyboard \
 combination (CTRL-COMMAND-SHIFT + R)."
 
 HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
-COPYRIGHT="2014-2017 Stefano Ceccherini"
+COPYRIGHT="2014-2018 Stefano Ceccherini"
 LICENSE="BSD (3-clause)
 	MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v${portVersion}.tar.gz"
-CHECKSUM_SHA256="373169858436b2aa626264911beed6f800011adbf443e9972e8af822fc17cf79"
-SOURCE_FILENAME="bescreencapture-${portVersion}.tar.gz"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="c38d0860e97ab2b4070a0d62c0ea554e09328601f253d6c46b87dede7485830f"
 SOURCE_DIR="bescreencapture-${portVersion}"
```

Comment:
Oops, my [previous message](https://github.com/haikuports/haikuports/pull/2411#discussion_r179114557) is hidden.
```diff
+SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
-SOURCE_DIR="bescreencapture-${portVersion}"
```
(The recipe will be ready for a merge after this change.)

Reply:
Done.
What's still missing in the recipe, is the placement of the Readme file. Is there an example t o look at, for this ?